src/current/_includes/v23.2/known-limitations/pcr-scheduled-changefeeds.md

```diff
@@ -0,0 +1 @@
+After the [cutover process]({% link {{ page.version.version }}/cutover-replication.md %}) for [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}), [scheduled changefeeds]({% link {{ page.version.version }}/create-schedule-for-changefeed.md %}) will continue on the promoted cluster. You will need to manage [pausing]({% link {{ page.version.version }}/pause-schedules.md %}) or [canceling]({% link {{ page.version.version }}/drop-schedules.md %}) the schedule on the original primary and promoted standby clusters to avoid two clusters running the same changefeed to one sink. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/123776)
```

Comment:
Note to docs team reviewers: the known limitation tracking GH issue link has different formats between v23.2 + v24.1 following the update to known limitations for GA.